kolibri/core/assets/src/views/breadcrumbs/index.vue

```diff
@@ -1,43 +1,64 @@
 <template>
 
-  <nav class="breadcrumbs">
-
-    <div v-if="collapsedCrumbs.length" class="breadcrumbs-dropdown-wrapper">
-      <ui-icon-button :has-dropdown="true" icon="expand_more" size="small">
-        <div slot="dropdown" class="breadcrumbs-dropdown">
-          <ol class="breadcrumbs-dropdown-items">
-            <li v-for="crumb in collapsedCrumbs" class="breadcrumbs-dropdown-item">
-              <router-link :to="crumb.link" :style="{ maxWidth: `${collapsedCrumbMaxWidth}px` }">
-                {{ crumb.text }}
-              </router-link>
-            </li>
-          </ol>
-        </div>
-      </ui-icon-button>
+  <div>
+    <nav class="breadcrumbs">
+      <div v-if="collapsedCrumbs.length" class="breadcrumbs-dropdown-wrapper">
```

Comment:
If I use `v-show`, there's an issue that occurs if you open the dropdown menu and the resize the screen, the dropdown button is hidden but the dropdwon menu still is a bit visible.